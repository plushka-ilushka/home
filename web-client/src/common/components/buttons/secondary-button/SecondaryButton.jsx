import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

import './secondary-button.scss';
import PrimaryButton from '../primary-button';

export default function SecondaryButton(props) {
  const { className, ...restProps } = props;

  return (
    <PrimaryButton
      className={classNames('secondary-button', className)}
      {...restProps}
    />
  );
}

SecondaryButton.propTypes = {
  className: PropTypes.string,
};

SecondaryButton.defaultProps = {
  className: undefined,
};
