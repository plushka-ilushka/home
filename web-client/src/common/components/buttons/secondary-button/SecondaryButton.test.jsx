import React from 'react';
import { mount } from 'enzyme';
import SecondaryButton from './SecondaryButton';

describe('NoPaintedButton Component', () => {
  it('NoPaintedButton Snapshot', () => {
    expect(mount(<SecondaryButton />)).toMatchSnapshot();
  });
});
