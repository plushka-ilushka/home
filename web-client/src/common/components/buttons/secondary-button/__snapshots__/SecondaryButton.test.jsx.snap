// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPaintedButton Component NoPaintedButton Snapshot 1`] = `
<SecondaryButton>
  <PrimaryButton
    className="secondary-button"
  >
    <Button
      className="primary-button secondary-button"
      type="button"
    >
      <button
        className="button primary-button secondary-button"
        type="button"
      />
    </Button>
  </PrimaryButton>
</SecondaryButton>
`;
